features/cascading_replication.feature

```diff
@@ -4,10 +4,10 @@ Feature: cascading replication
 Scenario: check a base backup from the replica
```

Comment:
Let's update the scenario to "check a base backup and streaming replication from a replica."